Makefile

```diff
@@ -36,8 +36,9 @@ help:
 
 
 build/%: DOCKER_BUILD_ARGS?=
+build/%: ROOT_CONTAINER?=ubuntu:22.04
```

Comment:
Slightly unrelated I suppose (I would be happy to merge this in a separate PR).

Reply:
Reverted.

Reply:
Added this to main as a separate commit.